utils/luci-app-ffwizard-berlin/root/www/cgi-bin/has_root-pass

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+echo "Content-type: text/plain"
+echo ""
+USER=root
+# taken from http://serverfault.com/a/240961
+grep ${USER} /etc/shadow | awk -F":" '{ if ($2 == "*" || $2 == "") \
```

Comment:
If I understand it correctly, "*" as password means the account is locked with no previous password set ever, so we shouldn't treat this as "login possible without password" either?